src/main/daml/Daml/Finance/Instrument/Bond/Util.daml

```diff
@@ -155,11 +155,15 @@ createCallableBondClaims dateToTime schedule callableSchedule periodicSchedule
                 else error "Only daily periods supported"
 
         coupon = give cpn
-        called = ("CALLED", principal)
-        notCalled = ("NOT CALLED", notCalledClaim)
-        tailClaim = if callingIsPossible then or called notCalled else notCalledClaim
+        principalInclCoupon = when (at p.adjustedEndDate) $ and coupon principal
+        notCalledClaimInclCoupon = when (at p.adjustedEndDate) $ and coupon notCalledClaim
```

Comment:
typically we don't abbreviate variable names, maybe use `notCalledClaimIncludingCoupon`